### PR TITLE
[1444] Rescue from 500s by showing the generic 500 page

### PR DESCRIPTION
**Pull request by tvararu**
### Context

Currently, the users get a blank page. This is an improvement.

### Guidance to review

The test I wrote resides in `view_pages_spec.rb ` which is probably not the best place?

### CODE REVIEW

**Comment by tvararu** [comment]
[Rescue from generic 500s with the existing 500 page](https://trello.com/c/buZhyJt3/1444-rescue-from-generic-500s-with-the-existing-500-page)

**Comment by misaka** [comment]
> Is there enough logging to denote where the user came from and where they going, are they logged in etc.

This should be done with request logging. Would be nice if AI could work.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception, except: :not_found
+  rescue_from StandardError, with: :internal_server_error
```
Comment (misaka) [inline app/controllers/application_controller.rb]: Does this still capture to Raven?

**Comment by misaka** [comment]
> The test I wrote resides in pages_controller_spec which is probably not the best place?

`spec/features/view_pages_spec.rb` where it is now seems reasonable enough.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (tvararu) [reply app/controllers/application_controller.rb]: It does not, I have addressed it in the method below, great spot!

**Comment by tvararu** [comment]
@defong this doesn't seek to improve the existing logging, just make sure that users see an appropriate error message while maintaining the same level of logging we do currently have. :)

**Pull request merged**